components/Register/Register.js

```diff
@@ -0,0 +1,35 @@
+import styled from "styled-components";
+const Register = () => {
```

Comment:
This should throw an error in eslint as arrow-body-style rule is applied in eslint config, and format the file on save